core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-category-list/dot-category-field-category-list.component.ts

```diff
@@ -54,7 +52,7 @@ export class DotCategoryFieldCategoryListComponent implements AfterViewInit {
     /**
      * Represents the variable 'categories' which is of type 'DotCategoryFieldCategory[][]'.
      */
-    categories = input.required<DotCategoryFieldCategory[][]>();
+    categories = input.required<DotCategoryFieldKeyValueObj[][]>();
```

Comment:
@oidacra We should use signal with `$`

Reply:
Leave it as it is today; we will change it later after finding a way to fix the test.